backend/src/main/java/com/map/service/UserService.java

```diff
@@ -27,4 +27,8 @@ public User createUser(UserDto userDto) {
     public List<User> getUsers() {
         return userRepository.getAllUsers();
     }
+
+    public void deleteUser(String userNickname) {
```

Comment:
You are using nickname in other places, it's nice to have this same here like:
`public void deleteUser(String nickname) {`